org.eclipse.wildwebdeveloper/language-servers/.gitignore

```diff
@@ -3,4 +3,5 @@ npm-debug.log
 /node/
 /etc/
 /*.tgz
-/server/
\ No newline at end of file
+/server/
+vscode-css-languageserver-1.0.0/
```

Comment:
Why do we need to add this ignore in .gitignore?

Reply:
Still investigating why this folder is created in language-servers.